src/balance.rs

```diff
@@ -110,26 +109,23 @@ pub fn balance(filename: &String) -> Result<(), std::io::Error> {
         println!(
             "  {0: <28} {1: <20}",
             account.account,
-            if account.amount < 0 {
-                (account.amount).to_formatted_string(&Locale::en).red().bold()
-            } else if account.amount == 0 {
-                (account.amount).to_formatted_string(&Locale::en).yellow().bold()
+            if account.amount < 0.0 {
+                format!("{0:.2}", account.amount).to_string().red().bold()
```

Comment:
You will see the `{0:.2}` in places and this is enforcing the two decimal place precision.